### PR TITLE
feat!: Update results endpoints

### PR DESCRIPTION
**Pull request by ospencer**

### CODE REVIEW

File: admin_requests.go
Comment (rnpridgeon) [inline admin_requests.go:84]: Do compute clients need to be able to support both v1 and v2? Typically I would say yes but since we are still in beta I'll leave that to your discretion.

**Review by rnpridgeon — DISMISSED** [review]
I'm not willing to block on this since we are in beta but I think the client should remain backward compatible post GA.  

We should also consider adding a CI workflow for running unit/integration tests and checkstyle

File: admin_requests.go
Comment (danielledeleo) [reply admin_requests.go:84]: If there's a need for v1 support an old version of this library can be used.

**Review by danielledeleo — DISMISSED** [review]

File: admin_requests.go
Comment (ospencer) [reply admin_requests.go:84]: For now, we're not worried about it, but we'll definitely care post 1.0 👍

Comment (rnpridgeon) [reply admin_requests.go:84]: so there is the very real possibility that someone will run an application in  a mixed version environment.

File: compute_test.go
Comment (rnpridgeon) [inline compute_test.go:144]: lets mock time in future PRs, time.Sleep is never the answer

**Review by MacklinEngineering — APPROVED** [review]

**Review by rnpridgeon — APPROVED** [review]
We need to look into mocking time for all timing related (Scheduled and such) events otherwise LGTM

File: compute_test.go
Comment (ospencer) [reply compute_test.go:144]: This is maybe the _one_ time it's the answer, since this is real time (these tests are full integration tests against a live instance).

**Pull request merged**
